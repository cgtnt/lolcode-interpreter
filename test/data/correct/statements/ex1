HAI

I HAS A c ITZ 5
I HAS A d ITZ 5

VISIBLE c d!

VISIBLE BOTH SAEM c AN d!
VISIBLE BOTH SAEM 0.0 AN 0.0!
VISIBLE SMOOSH "lol" "lmo" "word" MKAY!

KTHXBYE
