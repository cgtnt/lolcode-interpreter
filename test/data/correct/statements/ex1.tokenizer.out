@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_DECLARE_VAR:I HAS A-@3:T_T_IDENTIFIER:c-@3:T_DECLARE_SET_VAR:ITZ-@3:T_T_INT:5-@4:T_COMMAND_TERMINATOR:
-@4:T_DECLARE_VAR:I HAS A-@4:T_T_IDENTIFIER:d-@4:T_DECLARE_SET_VAR:ITZ-@4:T_T_INT:5-@6:T_COMMAND_TERMINATOR:
-@6:T_WRITE_STDOUT:VISIBLE-@6:T_T_IDENTIFIER:c-@6:T_T_IDENTIFIER:d-@6:T_BANG:!-@8:T_COMMAND_TERMINATOR:
-@8:T_WRITE_STDOUT:VISIBLE-@8:T_EQUAL:BOTH SAEM-@8:T_T_IDENTIFIER:c-@8:T_AND:AN-@8:T_T_IDENTIFIER:d-@8:T_BANG:!-@9:T_COMMAND_TERMINATOR:
-@9:T_WRITE_STDOUT:VISIBLE-@9:T_EQUAL:BOTH SAEM-@9:T_T_FLOAT:0.0-@9:T_AND:AN-@9:T_T_FLOAT:0.0-@9:T_BANG:!-@10:T_COMMAND_TERMINATOR:
-@10:T_WRITE_STDOUT:VISIBLE-@10:T_CONCAT:SMOOSH-@10:T_T_STRING:"lol"-@10:T_T_STRING:"lmo"-@10:T_T_STRING:"word"-@10:T_END_INF:MKAY-@10:T_BANG:!-@12:T_COMMAND_TERMINATOR:
-@12:T_END:KTHXBYE-@13:T_COMMAND_TERMINATOR:
-@13:T_EOF:
